src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryText.cs

```diff
@@ -111,7 +111,21 @@ public static FormulaValue Value(EvalVisitor runner, EvalVisitorContext context,
                 return new BlankValue(irContext);
             }
 
-            var (val, err) = ConvertToNumber(str, runner.CultureInfo);
+            // Checking if any culture was passed
+            var culture = runner.CultureInfo;
+            if (args.Length > 1)
+            {
+                if (args[1] is StringValue && CultureExists((args[1] as StringValue).Value))
```

Comment:
>if (args[1] is StringValue && CultureExists((args[1] as StringValue).Value)) [](http://example.com/codeflow?start=16&length=76)

nit:
You can get the var from the 'is' keyword: 
This lets you avoid is + as repeated. 

if (args[1] is stringValue str && 
   TryGetCulture(str,.Value, out culture))
{
  // error 
}

Reply:
Fixed it!